### ISSUE TITLE
Upgrading [to the WslFs file system format] fedoraremix with wsl/wslconfig fails

### ISSUE THREAD

**Issue by n3rd4i** [WhitewaterFoundry/Fedora-Remix-for-WSL#8]
**Describe the bug**
Trying to upgrade [WslFs] the fedoraremix distro with wsl or wslconfig fails.

**To Reproduce**
1. fedoraremix disto is installed
2. Execute command:
* `wslconfig /upgrade fedoraremix` (Microsoft Windows [Version 10.0.17763.195])
* `wsl --upgrade fedoraremix` (Microsoft Windows [Version 10.0.18309.1000])
4. See error:
``` sh
Upgrading...
Cannot upgrade extended attribute "user.Zif.MdChecksum[1546367604]".
Failed to upgrade "\\?\C:\Users\Andrei\AppData\Local\Packages\WhitewaterFoundryLtd.Co.FedoraRemixforWSL_s5fd8n49myqhm\LocalState\rootfs\var\cache\dnf\fedora-8c9e363c32277c01\packages\gpm-libs-1.20.7-16.fc29.x86_64.rpm": The parameter is incorrect.
Upgrading the distribution failed.
```

**Expected behavior**
Upgrade should be successful.

**Basic Troubleshooting Checklist**

[x] I have searched Google for the error message.
[x] I have checked official WSL troubleshooting documentation: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#confirm-wsl-is-enabled.
[x] I have searched the official Microsoft WSL issues page: https://github.com/Microsoft/WSL/issues.
[x] I have searched the WLinux issues page: https://github.com/WhitewaterFoundry/WLinux/issues.
[x] I have reset WLinux: Settings->Apps->Apps & features->WLinux->Advanced Options->Reset. 
[x] I have disabled and re-enabled WSL in Windows Features.
[x] I have run Windows 10 updates and restarted.

What other troubleshooting have you attempted?

Insert here: N/A

**WLinux Version**

Find: Settings->Apps->Apps & features->WLinux->Advanced Options->Version.

Insert here:
`10.0.19.0`

**Windows Build**

Run 'systeminfo | findstr /C:"OS"' in Command Prompt and insert here:
``` cmd
OS Name:                   Microsoft Windows 10 Pro Insider Preview
OS Version:                10.0.18309 N/A Build 18309
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
BIOS Version:              LENOVO 39CN16WW       , 7/29/2010
```

For help on retrieving: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#check-your-build-number

**Comment by DarthSpock** [comment]
ping

**Comment by gdzien** [comment]
in my case rm -Rf /var/cache/dnf/* have solved the problem and upgrade proceeded.

**Comment by delphym** [comment]
### "WslFs file system format" 
- what is it good for? 
- what is the other format(s) of distributions?
- what are the advantages having my WSL distro being upgraded to the WslFs file system format?

**Comment by gnw3** [comment]
See: https://github.com/MicrosoftDocs/WSL/issues/280. 
[TLDNR]: In the long run, WslFs will be the default.  There are bugs in the update process, and no real performance benefits for existing WSL installs.   

I want to try WslFs now in case there are issues for my use cases.    Debian and Ubuntu WSL apps both upgraded to WslFs "successfully", but Debian has some other problems -- I get a blank screen, can type on it but never get to the bash prompt that started with the update to Microsoft Windows [Version 10.0.18362.267].

**Comment by WSLUser** [comment]
It works with Pengwin. Granted the first time I tried entering cmd from Pengwin and it hung up but after a reboot and doing it from a pure elevated cmd prompt, it worked just fine (literally took seconds). FedoraRemix most likely will work just fine if you do what I did: reboot and open elevated CMD prompt then execute the wslfs upgrade command. I do recommend being on the latest Windows available (1903) before doing it though.
